### PR TITLE
complete but no README update

### PR DESCRIPTION
**Pull request by its-intricate**
methods such as land and take off are called on the plane object with the airport passed as an argument.
airport default capacity set as 10.

### CODE REVIEW

**Comment by deborahadebiyi** [comment]
Esther's code was easily readable and incorporated the single responsibility principle really well so I could easily identify what each method was doing. The code was DRY and commits were made at appropriate stages so I could easily identify a progression in the code.

**Review by deborahadebiyi — COMMENTED** [review]
Classes created in separate files which aids readability as well as a good use of attr reader and instance variables. Example messages are clear and give a good indication of what is being tested

**Review by deborahadebiyi — COMMENTED** [review]
Good use of edge cases and the code is very elegant. There is sufficient use of unit tests also

File: spec/plane_spec.rb [hunk 1]
```diff
@@ -0,0 +1,101 @@
+require 'plane'
+
+RSpec.describe Plane do
+  airport = Airport.new
+  airport1 = Airport.new
+  airport2 = Airport.new
+
+  it "lands at an airport" do
+    Airport.any_instance.stub(:weather) { false }
```
Comment (FabihaHassan123) [inline spec/plane_spec.rb:9]: Since  _Airport.any_instance.stub(:weather) { false } _ has been called a few times you can put this in a before block on line 7 and remove it from the places you have called it so that you don't have to repeat yourself.

File: spec/plane_spec.rb [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        Airport.any_instance.stub(:weather) { false }
+        airport = Airport.new(1)
+        plane = Plane.new
+        subject.land(airport)
+        message = 'Airport is full.'
+        expect { plane.land(airport) }.to raise_error(RuntimeError, message)
+      end
+
+      it "does not land if airport is stormy" do
+        Airport.any_instance.stub(:weather) { true }
```
Comment (FabihaHassan123) [inline spec/plane_spec.rb:34]: Keep _Airport.any_instance.stub(:weather) { true }_  in to override the before block :)

File: spec/plane_spec.rb [hunk 3]
```diff
@@ -0,0 +1,101 @@
+require 'plane'
+
+RSpec.describe Plane do
+  airport = Airport.new
+  airport1 = Airport.new
+  airport2 = Airport.new
+
+  it "lands at an airport" do
```
Comment (FabihaHassan123) [inline spec/plane_spec.rb:8]: You could put a _describe "#land" do_  before this to make it more readable.

File: spec/plane_spec.rb [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    Airport.any_instance.stub(:weather) { false }
+    subject.land(airport)
+    expect(airport.at_airport?(subject)).to eq true
+  end
+
+  it "takes off from an airport and confirms" do
```
Comment (FabihaHassan123) [inline spec/plane_spec.rb:14]: You can put a _describe "#takeoff" do_ and split the two expect lines into two different it blocks.

**Review by FabihaHassan123 — COMMENTED** [review]
Your code was really neat and organised! The methods were all concise and all your objects have a single responsibility which is great.
